<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiKinect TCPIP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_Viewer">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="CloudViewer" name="myCloudViewer" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_TCPIP">
   <property name="windowTitle">
    <string>TCPIP</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_closeall">
       <property name="text">
        <string>Close ALL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_openall">
       <property name="text">
        <string>Open ALL</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_ClientState">
       <property name="text">
        <string>Client : ?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_registrer">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_IP">
       <property name="text">
        <string>myIPlabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_graball">
       <property name="text">
        <string>Grab ALL</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Kinect">
   <property name="windowTitle">
    <string>Libfreenect</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>KinectWidget2</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="KinectWidget" name="myKinectWidget2" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>KinectWidget1</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="KinectWidget" name="myKinectWidget1" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBox_log">
       <item>
        <property name="text">
         <string>Debug</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Info</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Warning</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Error</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="LogWidget" name="logWidget_server" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KinectWidget</class>
   <extends>QWidget</extends>
   <header>KinectWidget.h</header>
   <container>1</container>
   <slots>
    <signal>PCGrabbedsignal(PointCloudT::Ptr)</signal>
    <slot>OpenKinect()</slot>
    <slot>CloseKinect()</slot>
    <slot>GrabKinect()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>CloudViewer</class>
   <extends>QWidget</extends>
   <header>CloudViewer.h</header>
   <container>1</container>
   <slots>
    <slot>showPC(PointCloudT::Ptr)</slot>
    <slot>setPointCloudPose(std::string,Transform)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
   <extends>QWidget</extends>
   <header>LogWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_openall</sender>
   <signal>clicked()</signal>
   <receiver>myKinectWidget1</receiver>
   <slot>OpenKinect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_openall</sender>
   <signal>clicked()</signal>
   <receiver>myKinectWidget2</receiver>
   <slot>OpenKinect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_closeall</sender>
   <signal>clicked()</signal>
   <receiver>myKinectWidget1</receiver>
   <slot>CloseKinect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_closeall</sender>
   <signal>clicked()</signal>
   <receiver>myKinectWidget2</receiver>
   <slot>CloseKinect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_graball</sender>
   <signal>clicked()</signal>
   <receiver>myKinectWidget1</receiver>
   <slot>GrabKinect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_graball</sender>
   <signal>clicked()</signal>
   <receiver>myKinectWidget2</receiver>
   <slot>GrabKinect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>myKinectWidget2</sender>
   <signal>PCGrabbedsignal(PointCloudT::Ptr)</signal>
   <receiver>myCloudViewer</receiver>
   <slot>showPC(PointCloudT::Ptr)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>884</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>myKinectWidget1</sender>
   <signal>PCGrabbedsignal(PointCloudT::Ptr)</signal>
   <receiver>myCloudViewer</receiver>
   <slot>showPC(PointCloudT::Ptr)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>862</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenAllDevices()</slot>
  <slot>CloseAllDevices()</slot>
  <slot>GrabAllDevices()</slot>
  <slot>TransformationChanged(TransformT)</slot>
 </slots>
</ui>
